Makefile

```diff
@@ -63,6 +63,8 @@ validate-latest-file:
 generate-validate-all: add-data-to-buckets
 	./scripts/generate_validate_all_netex.sh
 
+generate-email-odh-uploader:
+	./scripts/trigger_odh_uploader.sh periodMultiService
```

Comment:
Is there any way we can make this make command more flexible so it can be used for all netex types? Something along the lines of `./scripts/trigger_odh_uploader.sh $(file)` could work?

Reply:
Could we also possibly change this script so that it's purpose is to simply trigger the odh-uploader and log the email template created?